### PR TITLE
#160950885 Implementing feedback 

### PR DESCRIPTION
**Pull request by Bryan-Cee**
#### What does this PR do?
Implement feedback

#### Description of Task to be completed?
Add email to creating account details
Add quantity to ordering meals
Refactored tests

#### What are the relevant pivotal tracker stories?
[#160950885](https://www.pivotaltracker.com/story/show/160950885)

### CODE REVIEW

**Comment by mutoromike** [comment]
Change base branch to challenge 3

**Comment by Bryan-Cee** [comment]
> Change base branch to challenge 3

Done

File: app/api/v2/admin/models.py [hunk 1]
```diff
@@ -28,81 +27,43 @@ def add_to_menu(self, meal_name, meal_desc, meal_price):
...
-                                       'meal_desc': order[3],
-                                       'meal_price': order[4],
-                                       'order_status': order[5],
-                                       'time_of_order': order[6]})
-                    meal = ast.literal_eval(meal)
-                    meal['time_of_order'] = meal['time_of_order']['__value__']
-                    order_history.append(meal)
-
-                return make_response(jsonify({"The order": order_history}))
+                return make_response(jsonify({"The order": history}))
```
Comment (mutoromike) [inline app/api/v2/admin/models.py:69]: Good work removing the list thing.

File: app/api/v2/admin/views.py [hunk 2]
```diff
@@ -11,8 +11,8 @@
 @token_require
 def get_menu(current_user):
     if request.method == 'POST':
-        if not current_user[1]:
-            return jsonify({"Failed": "You are not an administrator"})
+        if not current_user['admin']:
+            return jsonify({"Failed": "You are not an administrator"}), 400
```
Comment (mutoromike) [inline app/api/v2/admin/views.py]: This should be a 401

File: app/api/v2/auth/views.py [hunk 3]
```diff
@@ -37,20 +39,20 @@ def login_user():
...
 
     with conn:
-        with conn.cursor() as cur:
+        with conn.cursor(cursor_factory=psycopg2.extras.RealDictCursor) as cur:
             cur.execute("SELECT user_id, username, password FROM Users WHERE username = %s", (authorization.username,))
             user = cur.fetchone()
 
             if user is None:
-                return make_response('Could not verify, invalid credentials', 401,
+                return make_response('Could not verify, invalid credentials check your username or password', 401,
```
Comment (mutoromike) [inline app/api/v2/auth/views.py]: This should be a 403

**Review by mutoromike — CHANGES_REQUESTED** [review]
Check the comments I have attached and review them.

**Comment by mutoromike** [comment]
This can now be merged

**Comment by mutoromike** [comment]
Good work refactoring the tests. This can now be merged.

**Pull request merged**